awx/api/serializers.py

```diff
@@ -4649,7 +4649,7 @@ class NotificationSerializer(BaseSerializer):
     class Meta:
         model = Notification
         fields = ('*', '-name', '-description', 'notification_template', 'error', 'status', 'notifications_sent',
-                  'notification_type', 'recipients', 'subject')
+                  'notification_type', 'recipients', 'subject', 'timeout')
```

Comment:
Probably don't need this, see comment below.

Reply:
👍